app/gather/assets/apps/survey/Survey.jsx

```diff
@@ -46,26 +65,62 @@ export default class Survey extends Component {
       total: props.survey.entities_count,
       labels: props.skeleton.docs,
       allPaths: paths,
-      selectedPaths: paths
+      selectedPaths: paths,
+      isConsumerActive: false,
+      activationError: null
     }
+    this.consumerToggle = this.consumerToggle.bind(this)
+  }
+
+  consumerToggle () {
```

Comment:
I think `verbNoun` is usually better than `nounVerb` for function names - `toggleConsumer` makes more sense